src/clusterfuzz/_internal/protos/uworker_msg.proto

```diff
@@ -35,7 +35,7 @@ message SetupInput {
 }
 
 message AnalyzeTaskInput {
-  repeated google.datastore.v1.Entity bad_builds = 1;
```

Comment:
This is backwards-incompatible. If there are any protos of this type already stored somewhere where they could be read later, or if messages of this type are already being exchanged over the network between two binaries that will be rolled out separately, then it would be better to mark 1 as reserved and increment the field index to 2:

```
reserved 1;

repeated int64 bad_revisions = 2;
```

If not, then this is fine!

Reply:
I don't think we have to worry about backward compatibility. @jonathanmetzman correct me if I'm wrong.

Reply:
It wouldn't matter for now. The tasks are all executed on one machine so as soon as they pull in the new source, they won't have any old protos.